tutorials/plot_artifacts_correction_ica.py

```diff
@@ -76,20 +76,19 @@
 # Component properties
 # --------------------
 #
-# Let's take a closer look at three potential candidates for artifact-related
-# components: IC 12, IC 15 and IC 21
+# Let's take a closer at properties of first three independent components
 
-# first, component 12:
-ica.plot_properties(raw, picks=12, dB=True)
+# first, component 0:
+ica.plot_properties(raw, picks=0)
 
 ###############################################################################
-# it looks like a blink component, but because the data were filtered
-# the spectrum plot is not very informative, let's change that:
-ica.plot_properties(raw, picks=12, dB=True, psd_args={'fmax': 35.})
+# we can see that the data were filtered so the spectrum plot is not
+# very informative, let's change that:
+ica.plot_properties(raw, picks=0, psd_args={'fmax': 35.})
 
 ###############################################################################
-# now let's inspect properties of components 15 and 21 (cardiac activity):
-ica.plot_properties(raw, picks=[15, 21], psd_args={'fmax': 35.})
+# we can also take a look at multiple different components at once:
+ica.plot_properties(raw, picks=[1, 2], psd_args={'fmax': 35.})
```

Comment:
Ok then it's fine. I think I just thought of the exclude/include params
that we also have here.
On Sat, 2 Jul 2016 at 19:41, Mikolaj Magnuski <notifications@github.com>
wrote:

> In tutorials/plot_artifacts_correction_ica.py
> <https://github.com/mne-tools/mne-python/pull/3379#discussion_r69381404>:
>
> >
> >  ###############################################################################
> > -# now let's inspect properties of components 15 and 21 (cardiac activity):
> > -ica.plot_properties(raw, picks=[15, 21], psd_args={'fmax': 35.})
> > +# we can also take a look at multiple different components at once:
> > +ica.plot_properties(raw, picks=[1, 2], psd_args={'fmax': 35.})
>
> Actually plot_sources uses picks for channels but all the other use picks
> for components.
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/mne-tools/mne-python/pull/3379/files/3aca2028c67b50aadeb9aa6f4fe4341e6c109deb#r69381404>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AB0fitvAPd34uuJD1nxLJssSebvQQKwVks5qRrDrgaJpZM4JDjjM>
> .
>